test/integration_test.py

```diff
@@ -387,16 +388,23 @@ def _list_managed_access_bundles(self,
         # prefix as possible.
         for source in self.managed_access_sources_by_catalog[catalog]:
             assert str(source.spec) in sources
-            bundle_fqid = self.random.choice(
-                self.azul_client.list_bundles(catalog, source, prefix='')
-            )
-            # FIXME: We shouldn't need to include the common prefix
-            #        https://github.com/DataBiosphere/azul/issues/3579
-            common = source.spec.prefix.common
-            prefix = bundle_fqid.uuid[len(common):8]
-            assert prefix != '', prefix
-            new_fqids = self.azul_client.list_bundles(catalog, source, prefix)
-            yield source, prefix, new_fqids
+            bundles = self.azul_client.list_bundles(catalog, source, prefix='')
```

Comment:
Don't like that this variable holding `list_bundle` results is called `bundles` while the other one is called `new_fqids`. 

```suggestion
            bundle_fqids = self.azul_client.list_bundles(catalog, source, prefix='')
```